scripts/flow/v10/chatListener.js

```diff
@@ -0,0 +1,22 @@
+import {getFlowInfo} from "./messageParser.js";
+import {isV10} from "../../utils.js";
+import {processFlowInfo} from "../common.js";
+
+
+const _onReady = () => {
+    if (!isV10()) return;
```

Comment:
This is v10 only. The v11 system no longer stores information in chat messages, so a different implementation is used.